src/components/global/componentWrapper.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import PropTypes from 'prop-types';
+import { Child } from './componentWrapper.styles';
+import GlobalStyle from '../globalStyle';
+import defaultTheme from '../themes';
+
+export const GlobalAndThemeWrapper = (props) => {
+  const { theme, children, isBold } = props;
+  if (children === undefined) {
+    return null;
+  }
+
+  return (
+    <ThemeProvider theme={theme}>
+      <>
+        <GlobalStyle />
+        <Child isBold={isBold}>
```

Comment:
I couldn't understand the reason of use this prop. the bold keyword is dedicated to typography components. it doesn't seem to be a general prop that should check in a wrapper component.

Reply:
I added it because of alt in the img tag, and we cant dedicate it on typography, in addition to that, we can use this wrapper for typography too, so get rid of these repetitive codes in each typography elements